bindings/cpp/src/lib.rs

```diff
@@ -1163,3 +1171,13 @@ pub fn new_eth_tx_info() -> ffi::EthTxInfoRaw {
         data: vec![],
     }
 }
+
+pub fn set_cronos_httpagent(agent: &str) -> Result<()> {
+    set_ethers_httpagent(agent)?;
+    Ok(())
+}
+
+pub fn get_cronos_httpagent() -> Result<String> {
+    let res = get_ethers_httpagent()?;
+    Ok(res)
+}
```

Comment:
no need for these bindings, just use the environment variable?

Reply:
yes, i'll remove it